manifests/dotfile.pp

```diff
@@ -24,7 +24,7 @@
 #
 define python::dotfile (
   Enum['absent', 'present'] $ensure   = 'present',
-  Stdlib::Filemode $filename          = $title,
+  Any $filename                       = $title,
```

Comment:
mhm, what about this:
```suggestion
  String[1] $filename                 = $title,
```

Reply:
I moved that fix to a separate PR: https://github.com/voxpupuli/puppet-python/pull/446